### ISSUE TITLE
eslint-plugin-jsx-a11y v5,6 causes linting failures due to airbnb-eslint incompatibilities

### ISSUE THREAD

**Issue by sjhworkman** [Yelp/beans#96]
Related to this github issue: [evcohen/eslint-plugin-jsx-a11y#232](https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/232)

Output Example:
```
ERROR in ./js/components/UserPreferenceForm.jsx

/Users/workman/beans/js/components/UserPreferenceForm.jsx
  1:1  error  Definition for rule 'jsx-a11y/href-no-hash' was not found  jsx-a11y/href-no-hash

✖ 1 problem (1 error, 0 warnings)
```

**Comment by kentwills** [comment]
magically fixed.
